docs/rules/custom/forbidden-function-call.md

```diff
@@ -0,0 +1,49 @@
+# forbidden-function-call
+
+**Summary**: Forbidden function call
+
+**Category**: Custom
+
+## Description
+
+This custom rule allows providing Regal a list of
+[built-in functions](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions) that should be
+considered forbidden. Any call to a function in the list will be considered a violation.
+
+Another, more advanced, option to achieve the same result is the
+[capabilities](https://www.openpolicyagent.org/docs/latest/deployments/#capabilities) feature in OPA. While a more
+capable option, it is also more demanding to configure and maintain. If you're already using the capabilities feature
+to forbid certain functions as part of your policy build pipeline, there's no need to enable this rule.
```

Comment:
```suggestion
to forbid certain functions as part of your policy build pipeline, there's no need to enable this rule unless you want to catch violations earlier in Regal.
```

Reply:
Yeah, I wasn't really thinking of CI here, but you're right that "build pipeline" implies that. You can pass `--capabilities` to `opa eval` or `opa test` locally too, of course.